topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: tutorial_hands_on
+
+title: Designing plasmids encoding predicted pathways by using the BASIC assembly method
+zenodo_link: https://zenodo.org/record/6123887#.YhTCPhvjJH4
+questions:
+- How to design plasmids encoding predicted pathways using BASIC assembly method ?
+- How to generate scripts enabling the automatic build of plasmids ?
+objectives:
+- Annotate the SBML pathway using Selenzy tool.
+- Design plasmids encoding predicted pathways using rpbasicdesign tool.
+- Generate dnabot scripts enabling the automatic build of plasmids.
+time_estimation: 15 min
+key_points:
+- Annotation of SBML pathway using Selenzyme
+- Generating Constructs using rpbasicdesign
+- Generating python scripts using dnabot
+contributors:
+- Kenza BAZI KABBAJ
+- Thomas DUIGOU
+- Joan HERISSON
+- Guillaume GRICOURT
```

Comment:
These need to be replaced with the keys you used in the contributors.yaml file, e.g. `kenza12` instead of the name.

Reply:
Ok, done.